public/app/plugins/datasource/loki/components/LokiLabelBrowser.tsx

```diff
@@ -124,11 +126,21 @@ const getStyles = (theme: GrafanaTheme2) => ({
     & + & {
       margin: ${theme.spacing(2, 0)};
     }
+
     position: relative;
   `,
+  footerSectionStyles: css`
+    padding: ${theme.spacing(1)};
+    background-color: ${theme.colors.background.primary};
+    position: sticky;
+    bottom: -${theme.spacing(3)}; /* offset the padding on modal */
+    left: 0;
+  `,
   selector: css`
     font-family: ${theme.typography.fontFamilyMonospace};
     margin-bottom: ${theme.spacing(1)};
+    overflow: scroll;
```

Comment:
I probably didn't pull correctly or something. Now when I am testing it, it sometimes gives me scroll and sometimes it breaks to a new line. What if we did `line-break: anywhere;` instead of scroll? Or keep a scroll and do `white-space: nowrap;` to not have a break at all. 

https://github.com/grafana/grafana/assets/30407135/5c0aa675-cd10-4445-824c-e6e41f017ce9

Reply:
Agreed, but I think I found a bug:
<img width="1728" alt="image" src="https://github.com/grafana/grafana/assets/109082771/3374ea21-ca25-4dbf-b218-40b748854e2c">
gdev-loki is giving different behavior then ops-dev-loki

I'll take a look after the all-hands

Reply:
Adding a pre tag seems to have fixed it, and looks a bit better
<img width="761" alt="image" src="https://github.com/grafana/grafana/assets/109082771/0c711515-ee04-4cd7-b9e6-2ec3df993882">

Reply:
It looks really good with the border!